### PR TITLE
Fix: sources/1stkissnovel.py

### PR DESCRIPTION
**Pull request by mchubby**
Based on fix for source/noveltranslate.py. Adds search.

### CODE REVIEW

**Comment by mchubby** [comment]
I copied them dumbly but I think the re and urllib imports are not required

**Comment by mchubby** [comment]
@SirGryphin If the PR is okay with you, the readme can mark this source as searchable.

**Comment by SirGryphin** [comment]
@mchubby thanks for fixing search, also the readme already has it marked as "searchable". The only thing you can do is update the readme.

**From this:**
`| https://1stkissnovel.love                    |     ✔      |           |      @SirGryphin      |`

**To this:**
`| https://1stkissnovel.love                    |     ✔      |           |@mchubby, @SirGryphin|`

**Comment by mchubby** [comment]
It's fine; I don't really mind not being listed as one of the maintainers :)
The source being very similar to other ones based on the same plugin like skynovel etc., it should be relatively easy to fix should it break.

File: lncrawl/sources/1stkissnovel.py [hunk 1]
```diff
@@ -1,81 +1,104 @@
...
     def download_chapter_body(self, chapter):
-        '''Download body of a single chapter and return as clean html format.'''
-        logger.info('Downloading %s', chapter['url'])
-        soup = self.get_soup(chapter['url'])
+        """Download body of a single chapter and return as clean html format."""
+        logger.info("Visiting %s", chapter["url"])
+        soup = self.get_soup(chapter["url"])
 
-        contents = soup.select_one('div.text-left')
+        contents = soup.select_one('div.text-left div:first-child')
```
Comment (dipu-bd) [inline lncrawl/sources/1stkissnovel.py:101]: this does not work on many chapters. e.g.: 

- https://1stkissnovel.love/novel/the-reborn-wife-is-too-seductive/chapter-9/
- https://1stkissnovel.love/novel/the-reborn-wife-is-too-seductive/chapter-10/

**Review by dipu-bd — CHANGES_REQUESTED** [review]

File: lncrawl/sources/1stkissnovel.py (hunk 1 above)
Comment (mchubby) [reply lncrawl/sources/1stkissnovel.py]: Good catch!

**Pull request merged**